app/templates/buyers/brief_overview.html

```diff
@@ -272,16 +272,13 @@ <h2 class="instruction-list-item-body">
     <div class="column-one-third">
       {% if brief.status == 'closed' %}
         <a href="{{ url_for('buyers.cancel_brief', framework_slug=framework.slug, lot_slug=brief.lotSlug, brief_id=brief.id) }}">Cancel requirement</a>
-     {% endif %}
+      {% elif brief.status == 'live' %}
+        <a href="https://www.gov.uk/guidance/how-to-make-changes-to-your-published-digital-outcomes-and-specialists-requirements#when-to-withdraw-your-requirements">Withdraw requirement</a>
+      {% elif framework.status == 'live' and brief.status == 'draft' %}
```

Comment:
Do we care if the framework is live or expired?

Reply:
Not sure! Is there any problem deleting draft requirements on an expired framework?

Reply:
No, suppliers can't apply for them at that point and because there will only be a max 2 week window for something to be draft on an expired framework before it becomes live..

Reply:
Cool - will remove.